src/components/Accordion/Root.tsx

```diff
@@ -0,0 +1,31 @@
+import {
+  Root as AccordionRoot,
+  Item as AccordionItem,
+} from "@radix-ui/react-accordion";
+import { ReactNode, useState } from "react";
+import { trigger } from "./common.css";
+
+export type AccordionProps = {
+  children: ReactNode;
+  defaultExpanded?: boolean;
+};
+
+const expandedValue = "accordion-value";
+
+export const Root = ({ children, defaultExpanded = false }: AccordionProps) => {
+  const [value, setValue] = useState(defaultExpanded ? expandedValue : "");
+
+  return (
+    <AccordionRoot
+      asChild
+      type="single"
+      collapsible
+      value={value}
+      onValueChange={setValue}
+    >
+      <AccordionItem value={expandedValue} className={trigger}>
+        {children}
+      </AccordionItem>
```

Comment:
question: what in case we want multiple AccordionItem, for example, FAQ section?

Reply:
Good question - I will extract to its own component for later 😄